api/metadata.go

```diff
@@ -82,24 +112,10 @@ func (api *DatasetAPI) getMetadata(w http.ResponseWriter, r *http.Request) {
 			return nil, err
 		}
 
-		// If dataset isn't published no 'Current' exists, use 'Next'
-		doc := datasetDoc.Current
-		if doc == nil {
-			if datasetDoc.Next == nil {
-				return nil, errors.New("invalid dataset doc: no 'current' or 'next' found")
-			}
-			doc = datasetDoc.Next
-		}
-
-		t, err := models.GetDatasetType(doc.Type)
-		if err != nil {
-			log.Error(ctx, "invalid dataset type", err, logData)
-			return nil, err
-		}
-
 		var metaDataDoc *models.Metadata
 
-		if t == models.CantabularBlob || t == models.CantabularTable || t == models.CantabularFlexibleTable || t == models.CantabularMultivariateTable {
+		if datasetType == models.CantabularBlob || datasetType == models.CantabularTable ||
```

Comment:
We dont use cantabularBlob or cantabularTable so these can be removed from the if statement to tidy it up